SCClassLibrary/Common/Control/Buffer.sc

```diff
@@ -341,7 +341,7 @@ Buffer {
 						startFrame = 0, leaveOpen = false, completionMessage;
 		// doesn't change my path
 		^["/b_write", bufnum, path,
-				headerFormat, sampleFormat, numFrames, startFrame,
+				headerFormat, sampleFormat, numFrames.asInt, startFrame.asInt,
```

Comment:
why not `asInteger`? `asInt` is more or less legacy AFAIK. 

I wonder in general: are there other places we'd need to do the same? This may end up in a lot of overhead in sclang for messages that are sent in large numbers.